js/src/collapse.js

```diff
@@ -331,7 +331,7 @@ const Collapse = (($) => {
         const _config = {
           ...Default,
           ...$this.data(),
-          ...typeof config === 'object' && config
+          ...typeof config === 'object' && config ? config : {}
```

Comment:
```js
typeof config === 'object' ? config : {}
```
?

Reply:
yep that can be enough thanks 👍

Reply:
Finally I won't make this change because it'll decrease our code coverage